requirements.txt

```diff
@@ -8,3 +8,4 @@ transformers[torch]==4.27.4
 # frontend only
 gradio==3.29.0
 grpcio==1.53.0
+orjson==3.8.7
```

Comment:
Thanks, Niklas.  Sorry I didn't get this in before you ran into it. Just go the fix confirmed yesterday.
Also I did a PR right before I saw this.  I'd like to give you author credit, but probably separate from the models change so I'm asking for your signoff info separately.